docs/reference/commandline/dockerd.md

```diff
@@ -3,6 +3,8 @@ title: "dockerd"
 aliases: ["/engine/reference/commandline/daemon/"]
 description: "The daemon command description and usage"
 keywords: "container, daemon, runtime"
+redirect_from:
+- /v1.11/engine/reference/commandline/daemon/
```

Comment:
Could you explain this redirect? Because the archive still exists at this location; https://docs.docker.com/v1.11/engine/reference/commandline/daemon/

Reply:
Our policy is to document all currently supported versions, plus the most recently EOL'd version (to support upgrades from that version), therefore we are removing all docs content for versions old than 17.03 to reduce the associated maintenance burden and customer confusion. These pages account for < 0.75% of current site traffic and are already not indexed by search engine. However out of an abundance of caution we are implementing a handful of redirects for the most popular pages within this <0.75%.  This PR implements those redirect for those pages.

Reply:
Have we tested if this actually works? Because I think it won't;

- These markdown files are not used for docs.docker.com (they have been replaced by stubs; https://github.com/docker/docker.github.io/tree/master/engine/reference/commandline that are propagated with the generated YAML files (see https://github.com/docker/docker.github.io/pull/7753)
- _If_ the redirect is created, we should check if it is not overwritten by the archive itself (https://github.com/docker/docker.github.io/blob/master/Dockerfile#L69-L83)


Also, once the "current" docs will be archived, possibly this redirect will now become;

```
/v18.09/v1.11/engine/reference/commandline/daemon/
```

Which will become a maintenance nightmare (also; what about version `/v1.12/`, `/v1.13/`)?

Would it be an alternative to, once we decide to no longer publish archives for a specific version, to replace an archive `/vYY.MM/*`  to redirect to `/` ? (or show a landing-page, informing the user that that specific archive is no longer published?)